net/tstun/wrap.go

```diff
@@ -522,7 +523,9 @@ type natV4Config struct {
 	// dstMasqAddrs is map of dst addresses to their respective MasqueradeAsIP
 	// addresses. The MasqueradeAsIP address is the address that should be used
 	// as the source address for packets to dst.
-	dstMasqAddrs views.Map[netip.Addr, netip.Addr] // dst -> masqAddr
+	dstMasqAddrs views.Map[key.NodePublic, netip.Addr] // dst -> masqAddr
+
+	routingTable *table.RoutingTable
```

Comment:
Could this be named more functionally than routing table? Which direction is this storing the routes for? (in vs out?) etc

Reply:
Maybe `nodeNativeIPTable` or something